app/controllers/application_controller.rb

```
@@ -9,7 +9,7 @@
 class ApplicationController < ActionController::API
   include Authorization
   include JwtToken
-  
+ 
```

RuboCop (1.86.1)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)